iib/workers/tasks/build.py

```diff
@@ -54,11 +61,14 @@
 
 
 @retry(
-    exceptions=ExternalServiceError,
-    tries=get_worker_config().iib_total_attempts,
-    delay=get_worker_config().iib_retry_delay,
-    jitter=get_worker_config().iib_retry_jitter,
-    logger=log,
+    before_sleep=before_sleep_log(log, logging.WARNING),
+    reraise=True,
+    retry=retry_if_exception_type(ExternalServiceError),
+    stop=stop_after_attempt(get_worker_config().iib_total_attempts),
```

Comment:
Maybe move the `get_worker_config()` call at the top of this file and use the same variable everywhere? That way, we only call the function once